flow/packet-headers/ipv6_routing.yaml

```diff
@@ -0,0 +1,525 @@
+            length: 1
+            default: 0
+            features:
+              - count
+          x-field-uid: 1
+        p_flag:
+          x-field-pattern:
+            description: Protected Flag. Indicates if the segment is protected
+              by a Fast Re-Route (FRR) mechanism.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              not_protected: 0
+              protected: 1
+            features:
+              - count
+          x-field-uid: 2
+        o_flag:
+          x-field-pattern:
+            description: OAM Flag. Indicates if the packet is an Operations, Administration,
+              and Maintenance (OAM) packet.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              not_oam: 0
+              oam: 1
+            features:
+              - count
+          x-field-uid: 3
+        a_flag:
+          x-field-pattern:
+            description: Alert Flag. Indicates the presence of important TLVs
+              that must be inspected.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              no_alert: 0
+              alert: 1
+            features:
+              - count
+          x-field-uid: 4
+        h_flag:
+          x-field-pattern:
+            description: HMAC Flag. Indicates the presence of the HMAC TLV for
+              verification.
+            format: integer
+            length: 1
+            default: 0
+            x-constants:
+              no_hmac: 0
+              hmac_present: 1
+            features:
+              - count
+          x-field-uid: 5
+        u2_flag:
+          x-field-pattern:
+            description: Unused 3 bits. Should be set to 0.
+            format: integer
+            length: 3
+            default: 0
+            features:
+              - count
+          x-field-uid: 6
+    Flow.Ipv6SegmentRouting.TlvPad:
+      description: Padding TLV used when more than one byte of padding is required
+        to align subsequent TLVs or pad the header to an 8-octet boundary.
+      type: object
+      properties:
+        type:
+          x-field-pattern:
+            description: Type identifier for PadN TLV. Always 4.
+            format: integer
+            length: 8
+            default: 4
+            features:
+              - count
+          x-field-uid: 1
+        length:
+          x-field-pattern:
+            description: The length of the Padding field in bytes. Can range from
+              0 to 5.
+            format: integer
+            length: 8
+            default: 0
+            features:
+              - count
+          x-field-uid: 2
+        padding:
+          description: Variable-length padding bytes. Must be set to 0 on transmission.
+          type: string
+          format: hex
+          default: '00'
+          x-field-uid: 3
+    Flow.Ipv6SegmentRouting.TlvHmac:
```

Comment:
If DUT tries to verify this feild should be generated by properly by means of what RFC says. So, how we are giving the input of pre-shared key, whether traffic will generate the HMAC accordingly?

key: The pre-shared key identified by HMAC Key ID
HMAC algorithm: Identified by the HMAC Key ID
Text: A concatenation of the following fields from the IPv6 header and the SRH, as it would be received at the node verifying the HMAC:

IPv6 header: Source address (16 octets)
SRH: Last Entry (1 octet)
SRH: Flags (1 octet)
SRH: HMAC 16 bits following Length
SRH: HMAC Key ID (4 octets)
SRH: All addresses in the Segment List (variable octets)

Reply:
HMAC TLV is removed from model.